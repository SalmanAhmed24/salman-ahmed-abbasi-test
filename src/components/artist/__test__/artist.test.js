import React from 'react';
import {  BrowserRouter } from 'react-router-dom';
import Artist from '../artist';
import "@testing-library/jest-dom/extend-expect";
import renderer from 'react-test-renderer';

test('<Artist> Component test',()=>{
    const component = renderer.create(<BrowserRouter>
                                        <Artist thumb_url='../../assets/logo.png' name='salman' facebook='https://facebook.com' />
                                    </BrowserRouter> );
    let tree = component.toJSON();
    
    expect(tree).toMatchSnapshot();

})