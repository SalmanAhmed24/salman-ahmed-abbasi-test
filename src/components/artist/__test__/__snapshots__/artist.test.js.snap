// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Artist> Component test 1`] = `
<div
  className="artist-main-wrap"
>
  <a
    className="card-link"
    href="/events/salman"
    onClick={[Function]}
  >
    <div
      className="main-card"
    >
      <div
        className="img-wrap"
      >
        <img
          alt=""
          className="img-card"
          src="../../assets/logo.png"
        />
      </div>
      <div
        className="card-info-wrap"
      >
        <div
          className="card-name"
        >
          <p>
            salman
          </p>
        </div>
        <div
          className="social-link-wrap"
        >
          <p>
            Facebook: 
            <span>
              https://facebook.com
            </span>
          </p>
        </div>
      </div>
    </div>
  </a>
</div>
`;
